ke-legacy/main.js

```diff
@@ -268,27 +268,26 @@ var ValNationalID = function(input){
     if (NumChar == 7 || NumChar == 8){return true;}
     else {return false;}
 };
-var IsPrePayTrialDistrict = function (){
-    return false;
-    //districtname = districtname.toLowerCase();
-    //if (districtname == "nyando" || districtname == "kipkelion" || districtname == "chwele"){return true}
-    //else {return false}
+
+var IsPrePayTrialDistrict = function(districtName){
+    return districtName === 'Kipkelion' || districtName === 'Chwele';
 };
+
 var GetPrepaymentAmount = function(client){
-    var ClientDistrict = client.DistrictName.toLowerCase();
-    var prepay = IsPrePayTrialDistrict(ClientDistrict);
-    if (prepay){
-        var PrePayTable = project.getOrCreateDataTable('PrePayment_IndividualAmounts');
-        var PrePayCursor = PrePayTable.queryRows({vars: {'accnum': client.AccountNumber}});
-        PrePayCursor.limit(1);
-        if (PrePayCursor.hasNext()) {
-            var PrePayRow = PrePayCursor.next();
-            return PrePayRow.vars.prepaymentamount;}
-        else {sendEmail('charles.lipeyah@oneacrefund.org', 'Prepayment amount not found', 'Prepayment amount not uploaded from client with acc num :'+ client.AccountNumber+ ' in district: '+client.DistrictName);
-            return 'Error';
-        }
-    }
-    else {return 500;}
+    return client.BalanceHistory[0].TotalCredit * 0.1;
+    // var prepay = IsPrePayTrialDistrict(client.DistrictName);
```

Comment:
won't this affect all the clients? the ticket says it should be  for only the 2 districts shouldn't this be
```suggestion
   var prepay = IsPrePayTrialDistrict(client.DistrictName);
   return prepay ?client.BalanceHistory[0].TotalCredit * 0.1 : 500;
```

Reply:
We are already doing the check in utils/menus/populateMenu.js
https://github.com/one-acre-fund/ussd-sms/blob/ce8d55035eb0b96e42fe46bee6eb2a371eaee3ee/ke-legacy/utils/menus/populate-menu/populateMenu.js#L8-L11